evaluation/langchain_trulens_full.ipynb

```diff
@@ -1,518 +1,518 @@
 {
```

Comment:
can you revert the changes in the `evaluation` folder?

Reply:
For the formatting changes, I'm using 
```
"[python]": {
  "editor.defaultFormatter": "ms-python.black-formatter"
}

```

Going to merge as is now since I can't revert those without reverting a bunch of stuff manually. Can normalize on one formatter for the team

Reply:
i meant reverting all the changes in the evaluation folder, not just formatting. 

nothing in that folder is run in CI or exposed to users at the moment.